jupyterlab_bigquery/jupyterlab_bigquery/handlers.py

```diff
@@ -75,7 +75,7 @@ def get_dataset_details(client, dataset_id):
   return {
       'details': {
           'id': "{}.{}".format(dataset.project, dataset.dataset_id),
-          'display_name': dataset.friendly_name,
+          'name': dataset.dataset_id,
```

Comment:
Following the BQ web UI, we display the `dataset_id` for datasets rather than `friendly_name`